docs/sources/flow/reference/components/loki.source.windowsevent.md

```diff
@@ -32,26 +32,29 @@ log entries to the list of receivers passed in `forward_to`.
-`use_incoming_timestamp` | `bool`               | When false, assigns the current timestamp to the log when it was processed.    | `false`                    | no
-`forward_to`             | `list(LogsReceiver)` | List of receivers to send log entries to.                                      |                            | yes
-`labels`                 | `map(string)`        | The labels to associate with incoming logs.                                    |                            | no 
+Name                     | Type                 | Description                                                                         | Default                    | Required
+------------------------ |----------------------|-------------------------------------------------------------------------------------|----------------------------| --------
+`locale`                 | `number`             | Locale ID for event rendering. 0 default is Windows Locale.                         | `0`                        | no
+`eventlog_name`          | `string`             | Event log to read from.                                                             |                            | See below.
+`xpath_query`            | `string`             | Event log to read from.                                                             | `"*"`                      | See below.
+`bookmark_path`          | `string`             | Keeps position in event log.                                                        | `"DATA_PATH/bookmark.xml"` | no
+`poll_interval`          | `duration`           | How often to poll the event log.                                                    | `"3s"`                     | no
+`exclude_event_data`     | `bool`               | Exclude event data.                                                                 | `false`                    | no
+`exclude_user_data`      | `bool`               | Exclude user data.                                                                  | `false`                    | no
+`exclude_event_message`  | `bool`               | Exclude the human-friendly event message.                                           | `false`                    | no
+`use_incoming_timestamp` | `bool`               | When false, assigns the current timestamp to the log when it was processed.         | `false`                    | no
+`forward_to`             | `list(LogsReceiver)` | List of receivers to send log entries to.                                           |                            | yes
+`labels`                 | `map(string)`        | The labels to associate with incoming logs.                                         |                            | no 
+`legacy_bookmark_path`   | `string`             | The location of the static bookmark path, will convert to flow compatible bookmark. | ``                         | no
```

Comment:
I think we want to say something here beyond `static bookmark path` which may not be obvious on what that means. Maybe a short description is needed up here and a more complete description down below? WDYT?

Reply:
We also should refer to Grafana Agent Flow in the same way... ie `flow` should be `{{< param "PRODUCT_NAME" >}}`